ajax/libs/sanctuary/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "sanctuary",
+  "description": "Refuge from unsafe JavaScript",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/sanctuary-js/sanctuary.git"
+  },
+  "dependencies": {
+    "sanctuary-def": "0.20.1",
+    "sanctuary-either": "1.2.0",
+    "sanctuary-maybe": "1.2.0",
+    "sanctuary-pair": "1.2.0",
+    "sanctuary-show": "1.0.0",
+    "sanctuary-type-classes": "11.0.0",
+    "sanctuary-type-identifiers": "2.0.1"
+  },
+  "devDependencies": {
+    "fantasy-land": "4.0.1",
+    "jsverify": "0.8.x",
+    "sanctuary-descending": "1.2.0",
+    "sanctuary-identity": "1.2.x",
+    "sanctuary-scripts": "4.0.x"
+  },
+  "files": [
+    "/LICENSE",
+    "/README.md",
+    "/index.js",
+    "/package.json"
+  ],
+  "filename": "bundle.min.js",
```

Comment:
Is this the correct value for `filename`? The (unminified) file in the repository is located at __dist/bundle.js__. My assumption is that `filename` must correspond to the name of the source file. Is this correct?

Reply:
As you've only specified `bundle.js` in the file-map, `filename` should be that. `filename` is the file that cdnjs will recommend as the default to use for the library.

Reply:
Thank you for your feedback, @MattIPv4!

> Note that you should have a `filename` to point to the main file of a lib, if that file is not minified, please still use `filename.min.js` or `filename.min.css` structure naming, we'll do the minify job.

Your suggestion appears to be at odds with the recommendation above, from __CONTRIBUTING.md__. Please confirm that `"bundle.js"` is the correct value.

Reply:
I'm not aware of any auto-minification we have running anymore. I'd suggest including your own distributed min file in the file-map if you have it. You are then welcome to choose either as the default file :)

Reply:
I have changed `filename` to `"bundle.js"`, and have opened #13665 to update __CONTRIBUTING.md__. I believe I have addressed all your feedback, @MattIPv4. :)